run.sh

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+BASE_DIR=$(pwd)
+CASE_FILE=$2
+#export RUST_LOG=debug
```

Comment:
Maybe add a `-v` argument that enables this?

Reply:
I will postpone this for now and open an issue.